modules/encrypted_cos_bucket/outputs.tf

```diff
@@ -42,62 +42,66 @@ output "bucket_storage_class" {
   value       = module.cos_bucket.bucket_storage_class
 }
 
+output "bucket_region" {
+  description = "Bucket region if you create a regional bucket"
+  value       = module.cos_bucket.bucket_region
+}
+
 output "kms_key_crn" {
   description = "The CRN of the KMS key used to encrypt the Object Storage bucket"
   value       = module.cos_bucket.kms_key_crn
 }
 
-output "cos_instance_id" {
-  description = "The ID of the Cloud Object Storage Instance where the buckets are created"
-  value       = module.cos_bucket.cos_instance_id
+output "instance_cbr_rules" {
+  description = "COS instance rules"
+  value       = module.cos_bucket.instance_cbr_rules
 }
 
-output "cos_instance_guid" {
-  description = "The GUID of the Cloud Object Storage Instance where the buckets are created"
-  value       = module.cos_bucket.cos_instance_guid
+output "cbr_rule_ids" {
+  description = "List of all rule ids"
+  value       = module.cos_bucket.cbr_rule_ids
 }
-
 output "bucket_cbr_rules" {
   description = "Object Storage bucket rules"
   value       = module.cos_bucket.bucket_cbr_rules
 }
+output "cos_instance_id" {
+  description = "The ID of the Cloud Object Storage Instance where the buckets are created"
+  value       = module.cos_bucket.cos_instance_id
+}
 
-output "instance_cbr_rules" {
-  description = "COS instance rules"
-  value       = module.cos_bucket.instance_cbr_rules
+output "cos_instance_guid" {
+  description = "The GUID of the Cloud Object Storage Instance where the buckets are created"
+  value       = module.cos_bucket.cos_instance_guid
 }
 
-output "cbr_rule_ids" {
-  description = "List of all rule ids"
-  value       = module.cos_bucket.cbr_rule_ids
+output "kms_crn" {
+  description = "CRN of the KMS instance when an instance"
+  value       = local.existing_kms_instance_crn
 }
 
 output "key_protect_guid" {
-  description = "Key Protect GUID"
-  value       = local.key_management_enabled ? module.key_protect_all_inclusive[0].kms_guid : null
+  description = "ID of the Key Protect instance which contains the encryption key for the object storage bucket"
+  value       = local.existing_kms_instance_guid
```

Comment:
should this be local.existing_kms_instance_crn ?

Reply:
In this case, no. I am exporting only the guid of the instance